api/HTMLAreaElement.json

```diff
@@ -214,10 +214,10 @@
               "version_added": true
             },
             "opera": {
-              "version_added": true
+              "version_added": "41"
```

Comment:
Confirmed with Opera 40+41 on BrowserStack using https://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement. I thought this seems too recent, but it's correct.